CefSharp.Core/ManagedCefBrowserAdapter.h

```diff
@@ -79,8 +79,8 @@ namespace CefSharp
             window.SetAsWindowless(hwnd, transparent);
             CefString addressNative = StringUtils::ToNative(address);
 
-            if (!CefBrowserHost::CreateBrowser(window, _clientAdapter.get(), addressNative,
-                *browserSettings->_browserSettings, NULL))
+            if (!CefBrowserHost::CreateBrowser(window, _clientAdapter.get(), addressNative,				
+                *browserSettings->_browserSettings, *browserSettings->RequestContext->_requestContext))
```

Comment:
Add an implicit `operator` so that you can cast RequestContext
https://github.com/amaitland/CefSharp/blob/feature/managedipc/CefSharp.BrowserSubprocess.Core/CefBrowserWrapper.h#L63